app/mixins/ads.js

```diff
@@ -97,12 +91,6 @@ export default Mixin.create({
 			adsData = this.get('adsData'),
 			globalFooter = document.querySelector('.wds-global-footer');
 
-		// TODO ADEN-6719 remove while removing MBLB leftovers
-		this.set(
-			'adsData.additionalConfig.MOBILE_PREFOOTER.disableManualInsert',
-			this.get('ads.module.adsContext.opts.isBLBOnMobileEnabled')
```

Comment:
Is it safe to unset this flag?

Reply:
`disableManualInsert` is hardcoded to `true` in config above
`opts.isBLBOnMobileEnabled` is set based on `wgAdDriverBottomLeaderBoardOnMobileCountries`
`wgAdDriverBottomLeaderBoardOnMobileCountries` is `[XX]` now

so `disableManualInsert` shouldn't be set to false here, so I think it is safe to remove this lines

Reply:
Thanks. Makes sense